sagemaker-python-sdk/mxnet_mnist/mxnet_mnist_with_batch_transform.ipynb

```diff
@@ -306,17 +287,42 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import ast\n",
+    "from numpy import genfromtxt\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "plt.rcParams['figure.figsize'] = (2,10)\n",
+    "\n",
+    "def show_digit(img, caption='', subplot=None):\n",
+    "    if subplot == None:\n",
+    "        _,(subplot) = plt.subplots(1,1)\n",
+    "    imgr = img.reshape((28,28))\n",
+    "    subplot.axis('off')\n",
+    "    subplot.imshow(imgr, cmap='gray')\n",
+    "    plt.title(caption)\n",
+    "\n",
+    "for i in range(10):\n",
+    "    input_file_name = 'data-{}.csv'.format(i)\n",
+    "    input_file_key = '{}/{}'.format(input_file_path, input_file_name)\n",
+    "    \n",
+    "    s3.Bucket(input_bucket_name).download_file(input_file_key, os.path.join(tmp_dir, input_file_name))\n",
```

Comment:
Get an error that `input_bucket_name` isn't defined.

Reply:
fixed